app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -6,6 +6,7 @@ import axios from 'axios';
 import Export from './Export';
```

Comment:
Similar to `Breadcrumb.js`, it would be nice to write a test suite for this file since there are a decent amount of changes to it

Reply:
Worked on this a little, but decided it's complex enough that I don't think I should start the test suite for this now. My changes are very minor in this component- mostly just adding a button to an existing button group.

Reply:
Sounds good to me, I'll document in my test coverage spreadsheet